src/libmscore/score.cpp

```diff
@@ -2750,6 +2750,46 @@ void Score::adjustKeySigs(int sidx, int eidx, KeyList km)
     }
 }
 
+//---------------------------------------------------------
+//   getKeyList
+//      This is taken from MuseScore::editInstrList()
```

Comment:
What does this mean, there is no such function?

Reply:
Not anymore, but there was such code in `instrdialog.cpp` in MuseScore 3.x. This dialog is replaced by a new one but this code is still required.